pyglossary/plugins/cc_cedict/__init__.py

```diff
@@ -0,0 +1,46 @@
+import re
+from pyglossary.plugins import formats_common
+from . import conv
+
+enable = True
+format = 'CC-CEDICT'
+# this typo is part of the API used by PyGlossary; don't change it!
+extentions = ('.u8',)
+
+entry_count_reg = re.compile(r'#! entries=(\d+)')
+class Reader:
+    def __init__(self, glos):
+        self.glos = glos
+        self.file = None
+        self.entries_left = 0
+
+    def open(self, filename, encoding='utf-8'):
+        if self.file:
+            self.file.close()
+        self.file = open(filename, 'r', encoding=encoding)
+        for line in self.file:
+            match = entry_count_reg.match(line)
+            if match is not None:
+                count = match.groups()[0]
+                self.entries_left = int(count)
+                break
+
+    def close(self):
+        if self.file:
+            self.file.close()
+        self.file = None
+        self.entries_left = 0
+
+    def __len__(self):
```

Comment:
Must return the total number of entries, or raise exception (like stardict.py) if has not opened glossary (and count is not available)